path/dynamic/dstarlite.go

```diff
@@ -117,6 +117,17 @@ func NewDStarLite(s, t graph.Node, g graph.Graph, h path.Heuristic, m WorldModel
 	return d
 }
 
+// edgeWeight is a helper function that returns the weight of the edge between
+// two connected nodes, u and v using the provided weight function. It panics
```

Comment:
The comma here looks strange to me, but I admit that rules for using commas in English are very unclear to me.